readme.md

```diff
@@ -230,6 +269,5 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+// This one is giving me a lot of trouble.  I prefer ruby/ActiveRecord to mongoose so mongoose is definitely something I need to keep up with moving forward.
```

Comment:
Definitely good to be working on. Going to want to push the item onto andy's wishlist array and save the model